samples/DistributedWebCrawler/DistributedWebCrawler.fs

```diff
@@ -128,6 +127,8 @@ let main orgargs =
         else
             Cluster.StartCluster( PrajnaClusterFile )
         let cluster = Cluster.GetCurrent()
+        let remote = DSet<_>(Name = "remoteExec")
+        remote.Execute(fun () -> ())
```

Comment:
This is sample code, I'd suggest removing these lines to avoid confusion or add some comments to help reader understand the code.